<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_username">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_login">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
